crates/sui-benchmark/src/drivers/bench_driver.rs

```diff
@@ -358,9 +362,13 @@ impl Driver<BenchmarkStats> for BenchDriver {
                                                 metrics_cloned.num_error.with_label_values(&[&b.1.get_workload_type().to_string(), "unknown_error"]).inc();
                                                 NextOp::Retry(b)
                                             }
-                                            Err(sui_err) => {
-                                                error!("{}", sui_err);
-                                                metrics_cloned.num_error.with_label_values(&[&b.1.get_workload_type().to_string(), &sui_err.to_string()]).inc();
+                                            Err(err) => {
+                                                if err.indicates_epoch_change() {
+                                                    reconfig(committee_cloned, qd_clone).await;
```

Comment:
Does the `committee_cloned` never need to be updated?

Reply:
it does need to be updated. fixed in my next commit